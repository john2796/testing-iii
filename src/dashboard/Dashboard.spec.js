// Test away
import React from 'react';
import { render, fireEvent, cleanup } from 'react-testing-library';
import renderer from 'react-test-renderer';

import Dashboard from './Dashboard';
// ### Dashboard
// - shows the controls and display

describe('<Dashboard />', () => {
  it('shows the controls and display', () => {
      const tree = renderer.create(<Dashboard />)
      expect(tree.toJSON()).toMatchSnapshot();
  })
})