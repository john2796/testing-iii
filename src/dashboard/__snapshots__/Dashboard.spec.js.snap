// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dashboard /> shows the controls and display 1`] = `
Array [
  <div
    className="display panel"
  >
    <div
      className="led green-led"
      data-testid="locked"
    >
      Unlocked
    </div>
    <div
      className="led green-led"
      data-testid="closed"
    >
      Open
    </div>
  </div>,
  <div
    className="controls panel"
  >
    <button
      className="toggle-btn"
      data-testid="closegate"
      disabled={true}
      onClick={[Function]}
      type="button"
    >
      Lock Gate
    </button>
    <button
      className="toggle-btn"
      data-testid="locked"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      Close Gate
    </button>
  </div>,
]
`;
